### ISSUE TITLE
ion-js v4.0.0; Maximum call stack size exceeded

### ISSUE THREAD

**Issue by jfuehner** [amzn/ion-js#601]
Hi! I'm running into the exception `Maximum call stack size exceeded` when trying to use the new DOM Api with v4.0.0. I am attempting to load an ion serialized `arrayBuffer` created via Java['s Jackson Ion serializer](https://github.com/FasterXML/jackson-dataformats-binary/tree/master/ion) which has an underlying object structure that looks like...

```
{
     data: {
          id: '123',
          featureCollections: [{
                id: '456',
                positions: [],
                fillColors: [],
                lineColors: []
          }]
     }
}
```

The `positions` (Double), `fillColors` (Integer) and `lineColors` (Integer) arrays contain ~ 1800000 values. 

```
Uncaught (in promise) RangeError: Maximum call stack size exceeded
    at new eval (Sequence.js?7237:9)
    at new List (List.js?0c8b:7)
    at _loadList (index.js?79f0:89)
    at _loadValue (index.js?79f0:71)
    at _loadStruct (index.js?79f0:82)
    at _loadValue (index.js?79f0:73)
    at _loadSequence (index.js?79f0:99)
    at _loadList (index.js?79f0:89)
    at _loadValue (index.js?79f0:71)
    at _loadStruct (index.js?79f0:82)
asyncGeneratorStep @ asyncToGenerator.js?1da1:6
_next @ asyncToGenerator.js?1da1:25
eval @ asyncToGenerator.js?1da1:32
eval @ asyncToGenerator.js?1da1:21
```

Thanks!

**Comment by zslayton** [comment]
Hi, thanks for reporting this! It looks like this is caused by [`Sequence` calling `Array#push` with too many arguments](https://github.com/amzn/ion-js/blob/a31484d87f09caad3a65b68163198bea1880f91b/src/dom/Sequence.ts#L21).

**Comment by zslayton** [comment]
This was fixed in #603.

**Comment by jfuehner** [comment]
@zslayton any chance this fix will be released soon? Thanks!

**Comment by zslayton** [comment]
I'll try to find time to release this next week. Are you able to work with the `HEAD` of `master` in the meantime, or is this blocking you?

**Comment by zslayton** [comment]
@jfuehner: This fix was included in [v4.0.1](https://github.com/amzn/ion-js/releases/tag/v4.0.1), which was released earlier today.
